components/AddToCartButton.js

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import {
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  View
+} from "react-native-web";
+
+export default function AddToCartButton(){
+
+  let total = '22.50'
+  // console.log(loading)
+  let loading = false
+
+    return (
+      <View  style={[styles.button, loading ? styles.disableButton  : styles.activeButton ]}>
+        <Text style={[styles.signupText, styles.buttonText]}>{loading ? 'Adding to Cart...' : `Add to Cart - $${total}`}</Text>
+      </View>
+    );
+};
+
+const styles = StyleSheet.create({
+  button: {
+    height: 55,
+    borderRadius: 30,
+    justifyContent: "center",
```

Comment:
lets default to using single quotes because there are always time where you need to add double quotes inside the text

Reply:
Ok. Make sense